src/objects/zcl_abapgit_object_iwmo.clas.abap

```diff
@@ -90,7 +90,37 @@ CLASS ZCL_ABAPGIT_OBJECT_IWMO IMPLEMENTATION.
 
   METHOD zif_abapgit_object~jump.
 
-    zcx_abapgit_exception=>raise( |TODO: Jump| ).
+    DATA: lv_mdl_technical_name TYPE /iwbep/med_mdl_technical_name,
+          lv_version            TYPE /iwbep/med_mdl_version,
+          lt_bdcdata            TYPE TABLE OF bdcdata,
+          ls_opt                TYPE ctu_params.
+
+    FIELD-SYMBOLS: <ls_bdcdata> LIKE LINE OF lt_bdcdata.
+
+    lv_mdl_technical_name = ms_item-obj_name.
+    lv_version = ms_item-obj_name+33(4).
+
+    APPEND INITIAL LINE TO lt_bdcdata ASSIGNING <ls_bdcdata>.
+    <ls_bdcdata>-program  = '/IWBEP/R_DST_MODEL_BUILDER'.
+    <ls_bdcdata>-dynpro   = '0100'.
+    <ls_bdcdata>-dynbegin = 'X'.
+    APPEND INITIAL LINE TO lt_bdcdata ASSIGNING <ls_bdcdata>.
+    <ls_bdcdata>-fnam = 'GS_MODEL_SCREEN_100-TECHNICAL_NAME'.
+    <ls_bdcdata>-fval = lv_mdl_technical_name.
+    APPEND INITIAL LINE TO lt_bdcdata ASSIGNING <ls_bdcdata>.
+    <ls_bdcdata>-fnam = 'GS_MODEL_SCREEN_100-VERSION'.
+    <ls_bdcdata>-fval = lv_version.
+
+    ls_opt-dismode = 'E'.
+    ls_opt-defsize = 'X'.
+
+    TRY.
+        CALL TRANSACTION '/IWBEP/REG_MODEL' WITH AUTHORITY-CHECK
+                                USING lt_bdcdata OPTIONS FROM ls_opt.
```

Comment:
Is `WITH AUTHORITY-CHECK` 702 compliant?

Reply:
Has someone a system to check it?

Reply:
I think its okay, https://syntax.abaplint.org/?filter=trans#/statement/CallTransaction